aten/src/ATen/native/native_functions.yaml

```diff
@@ -5322,16 +5322,44 @@
 - func: _nested_tensor_size(Tensor self) -> Tensor
   variants: method
   dispatch:
-    NestedTensorCPU, NestedTensorCUDA: NestedTensor_get_nested_size_tensor
+    NestedTensorCPU, NestedTensorCUDA: _nested_tensor_size
   autogen: _nested_tensor_size.out
 
+- func: _nested_tensor_strides(Tensor self) -> Tensor
+  variants: method
+  dispatch:
+    NestedTensorCPU, NestedTensorCUDA: _nested_tensor_strides
```

Comment:
Hmm, is it important for these to be dispatcher operators? (I'll read the quip to make sure I didn't miss anything)

Reply:
I don't want them to be dispatcher operators but they are needed for the backward of `.values` They are args to _nested_from_buffer. Unfortunately nested_tensor_size and the rest of the bunch are not accessible on Tensor.

Reply:
hmm is that a limitation of the autograd codegen? Why not have the derivative formula call `at::nested tensor::nested_tensor_strides(self)` instead of `self._nested_tensor_strides()`?

Or if it has to be a method, I'm wondering if it's less hassle to just directly write the method in `TensorBase.h` (and have a check to error out if the tensor isn't nested). One downside of making these native functions is that they get exposed to python (which is less bad since you're marking them as private, but still seems worse than making them C++ only).

Another artifact of making these into dispatcher ops is that if you were tracing code with nested tensors using `torch.fx.experimental.proxy_tensor.make_fx()`, you would see `_nested_tensor_strides` get directly baked into the trace as an ATen op (which, I'm not sure is what you want?)

Reply:
I think it is a limitation of the autograd codegen? At least thats what @albanD suggested to me would be the easiest way of exposing _nested_tenosr_strides() for derivatives.yaml. The TensorBase.h exposing is an interesting idea.

Sidenote I did add them to the `_SKIP_PYTHON_BINDINGS` to not exposing this to Python just yet